src/visualization/types/Map/view.tsx

```diff
@@ -222,6 +224,8 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
       ],
     }
   }
+
+  console.log('config: ', config)
```

Comment:
probably just want to git checkout this file since the changes are `console.log` statements